src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/updateindexmetadata/TransportUpdateManagedIndexMetaDataAction.kt

```diff
@@ -46,7 +41,6 @@ import org.elasticsearch.common.io.stream.Writeable
 import org.elasticsearch.index.Index
 import org.elasticsearch.threadpool.ThreadPool
 import org.elasticsearch.transport.TransportService
-import java.lang.Exception
 
 class TransportUpdateManagedIndexMetaDataAction : TransportMasterNodeAction<UpdateManagedIndexMetaDataRequest, AcknowledgedResponse> {
```

Comment:
Why does this still exist if we moved metadata to index?

Reply:
need this to clean old metadata in cluster state